loggly.go

```diff
@@ -92,7 +92,7 @@ func New(token string, tags ...string) *Client {
 // Send buffers `msg` for async sending.
 func (c *Client) Send(msg Message) error {
 	if _, exists := msg["timestamp"]; !exists {
-		msg["timestamp"] = time.Now().UnixNano() / int64(time.Millisecond)
+		msg["timestamp"] = time.Now().UTC().Format(time.RFC3339)
```

Comment:
The default format is RFC3339 with 9 digits of fractional second time (the RFC3339Nano format). The Loggly spec only supports 6 digits of fractional second time. I checked it myself and it only works with 6 digits.

I've added a commit to remove the UTC conversion and use a custom format string to match the supported format.